pxr/imaging/hdSt/interleavedMemoryManager.cpp

```diff
@@ -462,8 +462,11 @@ HdStInterleavedMemoryManager::_StripedInterleavedBuffer::Reallocate(
     // Skip buffers of zero size.
     if (totalSize > 0) {
         HgiBufferDesc bufDesc;
+        bufDesc.debugName = curRangeOwner_->GetResources().begin()->first.GetString();
         bufDesc.byteSize = totalSize;
-        bufDesc.usage = HgiBufferUsageUniform;
+        // Vulkan Validation layer is raising an error here because
+        // a usage flag of Uniform is applied to a buffer bound as storage
+        bufDesc.usage = HgiBufferUsageUniform | HgiBufferUsageStorage;
```

Comment:
If you want this to run with HgiGL, too, then can you add a Vulkan-specific check here like you did for vboMemoryManger.cpp?